etc/config/contextBroker

```diff
@@ -57,3 +57,10 @@ BROKER_DATABASE_NAME=orion
 # We need to start the CB with log append mode to not overwrite previous logs and logrotate work correctly
```

Comment:
An entry in the CHANGES_NEXT_RELEASE file should be added regarding this changes. Something like:

```
- Add: new GENERATE_COREDUMP variable at /etc/config/contextBroker to enable core generation and archiving (useful for debugging)
```

Reply:
Ops, I always forget the CNR.

Reply:
Fixed in 329a63aaa2f5df3dc1b8ee08ddfe908b1aa604fd and 1a7d031e866779bd1ee855b87375126f96b14851